src/test/java/org/sagebionetworks/web/client/presenter/PersonalAccessTokensPresenterTest.java

```diff
@@ -0,0 +1,76 @@
+package org.sagebionetworks.web.client.presenter;
+
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.times;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.runners.MockitoJUnitRunner;
+import org.sagebionetworks.web.client.GlobalApplicationState;
+import org.sagebionetworks.web.client.PlaceChanger;
+import org.sagebionetworks.web.client.place.PersonalAccessTokenPlace;
+import org.sagebionetworks.web.client.place.Profile;
+import org.sagebionetworks.web.client.place.Synapse;
+import org.sagebionetworks.web.client.view.PersonalAccessTokensView;
+
+import com.google.gwt.event.shared.EventBus;
+import com.google.gwt.junit.GWTMockUtilities;
+import com.google.gwt.user.client.ui.AcceptsOneWidget;
+import com.google.gwt.user.client.ui.Widget;
+
+@RunWith(MockitoJUnitRunner.class)
+public class PersonalAccessTokensPresenterTest {
+
+    @Mock
+    PersonalAccessTokensView mockView;
+
+    @Mock
+    GlobalApplicationState globalApplicationState;
+
+    @InjectMocks
+    PersonalAccessTokensPresenter presenter;
+
+    @Test
+    public void testSetPlace() {
+        PersonalAccessTokenPlace place = new PersonalAccessTokenPlace("token");
+
+        // Method under test
+        presenter.setPlace(place);
+
+        // Called once in the constructor and once on setPlace invocation
+        verify(mockView, times(2)).setPresenter(presenter);
+    }
+
+    @Test
+    public void testStart() {
+        GWTMockUtilities.disarm();
```

Comment:
Disarming due to the "new Widget()"?  If that's the reason, you could instead have the when(mockView.asWidget()) return a mock IsWidget.class to avoid this issue.  Then verify verify(mockPanel).setWidget is called with this mock object.

Reply:
`asWidget` must return a `Widget` (not an `IsWidget`). I can `mock(Widget.class)` instead of `new Widget()`, but that still requires calling disarm.

Reply:
That's right.  We must not be verifying the actual object in other tests.  Carry on 👍